.pre-commit-config.yaml

```diff
@@ -61,13 +61,14 @@ repos:
         name: mypy
         entry: script/run-in-env.sh mypy
         language: script
-        types_or: [python, pyi]
         require_serial: true
+        types_or: [python, pyi]
         files: ^(homeassistant|pylint)/.+\.(py|pyi)$
       - id: pylint
         name: pylint
-        entry: script/run-in-env.sh pylint -j 0 --ignore-missing-annotations=y
+        entry: script/run-in-env.sh pylint --ignore-missing-annotations=y
```

Comment:
This is inefficient for our CI. We should make sure our CI does run as fast as possible.

Reply:
GitHub CI does not use this config.  It uses the config here, which is already configured to run without parallel execution:
https://github.com/home-assistant/core/blob/dev/.github/workflows/ci.yaml#L710

Do you have other CI systems which do use `.pre-commit-config.yaml`?